### PR TITLE
Suggest visual element to identify automation windows and other security clarifications

### PR DESCRIPTION
**Pull request by andreastt**
This change is [Reviewable](https://reviewable.io/reviews/w3c/webdriver/335)

### CODE REVIEW

File: webdriver-spec.html [hunk 1]
```diff
@@ -7429,21 +7429,42 @@
...
+ to test whether to enable WebDriver,
+ or alternatively make the user agent initiate
+ or confirm the connection through
+ a privileged content document or control widget,
+ in case the user agent does not
  directly implement the HTTP endpoints.
 
-<p>To prevent arbitrary machines from connecting and
- creating <a>sessions</a>, it is suggested that only connections from
+<p>It is strongly suggested that user agents
```
Comment (shs96c) [inline webdriver-spec.html]: Not mentioned in the commit message, but makes sense.

File: webdriver-spec.html [hunk 2]
```diff
@@ -7429,21 +7429,42 @@
...
+ CIDR range <code>127.0.0.0/8</code>
+ and the IPv6 localhost address <code>::1</code>. [[!RFC4632]]
+
+<p>It is also suggested that user agents
+ make an effort to visually distinguish
+ a user agent session that is under control of WebDriver
+ from those used for normal browsing sessions.
+ This can be done through a chrome UX element
+ such as a “door hanger”,
+ colourful decoration of the OS window,
```
Comment (shs96c) [inline webdriver-spec.html]: I believe we use US English in the spec. "colorful"

**Comment by shs96c** [comment]
I have no idea if it's possible to edit this inline.

File: webdriver-spec.html (hunk 2 above)
Comment (andreastt) [inline webdriver-spec.html]: Fixed.

File: webdriver-spec.html (hunk 1 above)
Comment (andreastt) [inline webdriver-spec.html]: Good point. Added it to the commit message, but I’m happy to split it out as a separate commit if you think it is best.

**Comment by AutomatedTester** [comment]
landed in a8ac274 et al
